### ISSUE TITLE
Influx 0.9 rc31 UI Issues in admin interface

### ISSUE THREAD

**Issue by dublx** [influxdb/influxdb#2719]
Hi, 
Using Influx 0.9 rc31 in Mac (via Homebrew) I get some strange behaviour on the UI. 
Using Chrome I dont get the same layout/fields as shown in the Getting Started:
-After creating a DB, I select Explore Data and in the Write Data section there is only a big textbox to put in the JSON, but no input box for the Point name as shown in the tutorial.

-On the manage user page the following error is displayed in the Dev tools console, which I think prevents loading the user and subsequently changing it (eg admin on/off) :
```javascript
"Error: window.influxdb.getDatabaseUser is not a function
e.getDatabaseUser@http://localhost:8083/javascripts/all.js:33:26173
e.showDatabaseUser@http://localhost:8083/javascripts/all.js:33:27480
mo.prototype.functionCall/<@http://localhost:8083/javascripts/all.js:29:10824
ia[t]</<.compile/</</i@http://localhost:8083/javascripts/all.js:29:17265
Gn/this.$get</f.prototype.$eval@http://localhost:8083/javascripts/all.js:28:2390
Gn/this.$get</f.prototype.$apply@http://localhost:8083/javascripts/all.js:28:2615
ia[t]</<.compile/</<@http://localhost:8083/javascripts/all.js:29:17315
ot.event.dispatch@http://localhost:8083/javascripts/all.js:25:1984
ot.event.add/v.handle@http://localhost:8083/javascripts/all.js:24:31582
" all.js:27```

**Comment by beckettsean** [comment]
The screenshots in the 0.9 documentation appear to have been kept from the 0.8 docs and are not quite the same. There is no point name to enter for 0.9. I've opened an issue to get this fixed. influxdb/influxdb.org#139

I can repro the error in the JS console and also confirm that the changes don't persist.

**Comment by toddboom** [comment]
this was removed intentionally for v0.9.0. now that there are both tags and fields, the admin interface allows writing only raw JSON and doesn't do any object conversion for you, thus the name field is no longer applicable.

the current admin interface will also be replaced with a new version for v0.9.0 final, so expect more changes before then.

**Comment by toddboom** [comment]
This should no longer be an issue once #2746 is merged. It should be included as part of the v0.9.1 release.
